demos/projects/ESPRESSIF/aziotkit/components/azure-iot-kit-sensors/src/oled.c

```diff
@@ -0,0 +1,45 @@
+/* OLED screen Example
+
+   For other examples please check:
+   https://github.com/espressif/esp-iot-solution/tree/master/examples
+
+   This example code is in the Public Domain (or CC0 licensed, at your option.)
+
+   Unless required by applicable law or agreed to in writing, this
+   software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+#include <stdio.h>

```

Comment:
is this also required?

Reply:
these come from espressif, probably should not modify

Reply:
https://github.com/espressif this does not contains these files.